bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableClusterUtilities.java

```diff
@@ -217,7 +328,7 @@ public synchronized int getClusterNodeCount(String clusterId, String zoneId) {
    * @return the {@link Cluster} if it was set. If the cluster is not found, throw a {@link
    *     NullPointerException}.
    */
-  public synchronized Cluster getCluster(String clusterId, String zoneId) {
+  public Cluster getCluster(String clusterId, String zoneId) {
     Cluster response = null;
     for (Cluster cluster : getClusters().getClustersList()) {
```

Comment:
We allow users to set only the projectId.  This method allows us to use v1 configuration with a v2 client.  Let's speak offline about this.